xinference/model/embedding/tests/test_integrated_embedding.py

```diff
@@ -36,3 +38,24 @@ def test_sparse_embedding(setup):
     words = model.convert_ids_to_tokens(token_ids)
     assert len(words) == len(token_ids)
     assert isinstance(words[0], str)
+
+
+def test_clip_embedding(setup):
+    endpoint, _ = setup
+    client = Client(endpoint)
+
+    model_uid = client.launch_model(
+        model_name="jina-clip-v2", model_type="embedding", torch_dtype="float16"
+    )
+    assert len(client.list_models()) == 1
+
+    model = client.get_model(model_uid)
+    image_str = "https://i.ibb.co/r5w8hG8/beach2.jpg"
+    input = ["This is a picture of diagram", "a dog", "海滩上美丽的日落。", image_str, image_str]
```

Comment:
Can you add a test case about the base64 image, and you can put it at the first or second place.